### PR TITLE
docs: add KeyboardToolBarTheme type example

### PR DESCRIPTION
**Pull request by RodSarhan**
## 📜 Description

Added theme type to KeyboardToolbar theme example

## 📝 Checklist

- [x] CI successfully passed
- [x] I added new mocks and corresponding unit-tests if library API was changed

### CODE REVIEW

**Comment by kirillzyusko** [comment]
Hey @RodSarhan 👋 

Thank you for the PR, but can't you extract `KeyboardToolbarTheme` from `KeyboardToolbarProps` as `KeyboardToolbarProps["theme"]`?

**Comment by RodSarhan** [comment]
Hello, thanks for this awesome package 😁

Yeah it's not a big deal, I thought you just accidentally forgot to export it since most of the other useful types are already there.

**Comment by kirillzyusko** [comment]
@RodSarhan I think it was kind of intentional, because I didn't want to export many types from this package and if something belongs to a particular component (like a particular prop) then it should be retrieved from corresponding `Props` type.

Maybe you can change documentation to show how to extract `KeyboardToolbarTheme` from `KeyboardToolbarProps`?

image

Will be happy to review such PR (you can do these changes in this PR) 😊

File: docs/docs/api/components/keyboard-toolbar/index.mdx [hunk 1]
```diff
@@ -215,9 +215,10 @@ A boolean prop indicating whether to show `next` and `prev` buttons. Can be usef
 Prop allowing you to specify the brand colors of your application for `KeyboardToolbar` component. If you want to re-use already platform specific colors you can import `DefaultKeyboardToolbarTheme` object and override colors only necessary colors:
 
 ```ts
-import { DefaultKeyboardToolbarTheme } from "react-native-keyboard-controller";
+import { DefaultKeyboardToolbarTheme, type KeyboardToolbarProps } from "react-native-keyboard-controller";
```
Comment (kirillzyusko) [inline docs/docs/api/components/keyboard-toolbar/index.mdx:218]: I didn't add `type` keyword in documentation (typically if people are using it, then they have eslint that automatically does correction) - let's be consistent across documentation and remove `type` from import statement 🙏

File: docs/docs/api/components/keyboard-toolbar/index.mdx [hunk 2]
```diff
@@ -215,9 +215,10 @@ A boolean prop indicating whether to show `next` and `prev` buttons. Can be usef
...
+type KeyboardToolbarTheme = KeyboardToolbarProps["theme"]
 
-const theme = {
+const theme: KeyboardToolbarTheme = {
```
Comment (kirillzyusko) [inline docs/docs/api/components/keyboard-toolbar/index.mdx:221]: Maybe better to inline type without creating a new type? Like:

```tsx
const theme: KeyboardToolbarProps["theme"] = {
```

**Comment by kirillzyusko** [comment]
@RodSarhan I left two minor improvements/suggestions - when you make changes I'll be happy to merge them 😊

**Comment by RodSarhan** [comment]
got it, @kirillzyusko Thanks for reviewing, and sorry for bothering you with such a small detail

**Comment by kirillzyusko** [comment]
> Thanks for reviewing, and sorry for bothering you with such a small detail

Everything is okay, no worries 😊

**Comment by kirillzyusko** [comment]
@RodSarhan would you mind to fix prettier? 👀 `npm run format` from root of the project

**Comment by kirillzyusko** [comment]
Thank you for your contribution @RodSarhan ❤️

**Pull request merged**